packages/strapi-plugin-users-permissions/controllers/Auth.js

```diff
@@ -315,29 +315,21 @@ module.exports = {
       key: 'advanced',
     });
 
+    const userInfo = _.omit(user, ['password', 'resetPasswordToken', 'role', 'provider']);
+
```

Comment:
Is it just a refacto or there is a security fix I don't see?

Reply:
nope just a cleanup I made because I was doing the fix there in the first place.